docs/architecture/gsp-architecture-infrastructure.md

```diff
@@ -7,18 +7,28 @@ __[edit draw.io diagram](https://www.draw.io/?state=%7B%22ids%22:%5B%221hUinA_Be
 -->
 
 
-1. The GSP cluster resides in a single AWS account within the London region (__eu-west-2__)
+1. A GSP cluster resides in an AWS account within the London region (__eu-west-2__)
 
-2. The infrastructure is deployed across three availability zones __eu-west-2a__, __eu-west-2b__, __eu-west-2c__
+2. The infrastructure is deployed across three availability zones (__eu-west-2a__, __eu-west-2b__, __eu-west-2c__)
 
-3. The state for the cluster is persisted across three etcd nodes in different availability zones.
+3. The cluster makes use of [Global Accellerator](https://aws.amazon.com/global-accelerator/) to manage static IP addresses.
 
-4. By default there are three masters for availability and resilience but this can be overridden.
+4. Load balancing is achieved through the use of an [application load balancer](https://aws.amazon.com/elasticloadbalancing/features/#Details_for_Elastic_Load_Balancing_Products).
 
-5. By default there are three worker nodes for the cluster split across the three availability zones
+5. External egress access is via a [NAT gateway](https://docs.aws.amazon.com/vpc/latest/userguide/vpc-nat-gateway.html) in each availability zone.
 
-6. The cluster relies on AWS IAM for identity and authorisation.
+6. The [kubernetes control plane](https://kubernetes.io/docs/concepts/#kubernetes-control-plane) is managed by [AWS EKS](https://aws.amazon.com/eks)
 
-7. All accounts benefit from AWS Shield protection against DDOS attack
+7. The cluster relies on [AWS IAM](https://aws.amazon.com/iam) for identity and authorisation
 
-8. The cluster aggregates log events to CloudWatch for ongoing processing by the Cyber Security team
+8. The cluster uses [kiam](https://github.com/uswitch/kiam) to allow cluster users to associate [AWS IAM](https://aws.amazon.com/iam) roles to kubernetes pods
+
+9. The cluster may contain an optional autoscaling group containing a continuous integration service based on [ConcourseCI](http://concourse.ci/)
+
+10. By default there are three [kubernetes worker nodes](https://kubernetes.io/docs/concepts/architecture/nodes/) for the cluster split across the three availability zones
+
+11. All accounts benefit from [AWS Shield](https://aws.amazon.com/shield/) protection against distributed denial of service attacks
+
+12. The cluster aggregates log events to [AWS CloudWatch](https://aws.amazon.com/cloudwatch/) for ongoing processing by the Cyber Security team
+
+13. CloudWatch logs are shipped externally to Splunk using [AWS Lambda](https://aws.amazon.com/lambda/)
```

Comment:
I don't think that's true. Only specific logs or the ones that are security risk are shipped to splunk.

Reply:
Also I'm not certain Lambda should be mentioned... /cc @chrisfarms

Reply:
Only the "pod" logs are currently forwarded to splunk. There are rough plans to not use a lambda for this forwarding in the future and do something like AWS Firehose instead.

Reply:
lets change "aggregates log events" to "aggregates selected log events". I'm minded to leave the reference to lambda in since it is how it currently works now, if it changes in the future we can update it to reflect any changes. Think we also need to add a splunk box